kitchen/README.md

```diff
@@ -28,3 +28,26 @@ $ curl -v -XPOST -H "x-kitchen-cookies: a=b,c=d" $(hostname):PORT
 < Set-Cookie: c=d
 ...
 ```
+
+# Demos
+
+## Image Search and Tagging
+
+The demo application performs image search and then tags the images using the inception-v3 model built in tensorflow.
+The demo highlights the following features of waiter:
+1. Invoking different services from a client (i.e. the demo page)
+1. Running different services (image search and image tagging) on waiter
+1. Services with different concurrency level support
+1. Scaling of services (image tagging)
```

Comment:
Can we add a hyperlink to https://github.com/twosigma/waiter/blob/master/waiter/docs/autoscaling.md#waiters-autoscaling-algorithm for "Scaling of services"?

Reply:
Done.